### DISCUSSION TITLE
Debian Bullseye, KDE, 3 PCs, AdBlock Plus and saving downloads does not work, dbus error

### DISCUSSION THREAD

**Discussion by MarekAP** [Eloston/ungoogled-chromium#1762] [General]
I have 3 PC.
2 are Intel based (2nd and 4th generation) and other is AMD based.
AdBlock Plus have had to be disabled on all of them, as it was blocking everything and no websites were loading.
Saving downloads works only on desktop, which is AMD FX based (don't judge, I needed a lot of wide PCIe and no other platform had enough, but ThreadRipper).
No Intel based PC does save any download, save windows appears, but it never saves.

I am getting such error
ERROR:select_file_dialog_impl_portal.cc(698)] Portal returned error: org.freedesktop.DBus.Error.NoReply:

Just to clarify. I have exactly same packages on all 3 PCs. The only difference are graphics drivers and some packages from outside Debian like signal.

I have used binary package from repository for Debian Bullseye.

**Comment by MarekAP** [comment]
I have compared both /etc/dbus-1/ and /usr/share/dbus-1/ on laptop and desktop and those are same (hash of list of hashes of all files is same). It doesn't seem to be caused be either difference in packages or dbus differences between hosts.
I am lost for solution

**Reply by networkException** [reply]
Can you reproduce the issue with a clean profile?

**Reply by MarekAP** [reply]
I have tried both incognito and after moving both .config/chromium and .cache/chromium (like chromium never was installed previously). I tried on different user account as well (with existing profile and with profile moves away). Just made new account, tried and failed again

**Reply by MarekAP** [reply]
I meant that downloads doesn't work in all situations described above. I downgraded to 93 and it works (I have not apt clean my lady's Lenovo, so could scp it)

**Reply by networkException** [reply]
I'm a bit clueless, I would recommend to try a different packaging like flatpak in the meanwhile and see if updates to the system or chromium resolves this issue eventually. I've not seen something like this happen before and the error message suggests a problem with the desktop rather than chromium itself.

**Reply by MarekAP** [reply]
I usually am able to sort such issues myself. This time I am clueless too.
I tried flatpak, but whole idea of flatpak is not for me any longer (7 years ago it would be fine, now I just want working system, no tinkering with basic functionality).
I would try to find something on snap, as it seem easier to find binaries and run.
